webapp/src/client/client.js

```diff
@@ -9,10 +9,10 @@ import Constants from '../constants';
 export default class Client {
     constructor() {
         const url = new URL(window.location.href);
-        this.baseUrl = `${url.protocol}//${url.host}`;
-        this.pluginUrl = `${this.baseUrl}/plugins/${Constants.PLUGIN_NAME}`;
-        this.apiUrl = `${this.baseUrl}/api/v4`;
-        this.pluginApiUrl = `${this.pluginUrl}/api/v1`;
+        this.baseUrl = url.protocol + '//' + url.host;
+        this.pluginUrl = this.baseUrl + '/plugins/' + Constants.PLUGIN_NAME;
+        this.apiUrl = this.baseUrl + '/api/v4';
+        this.pluginApiUrl = this.pluginUrl + '/api/v1';
```

Comment:
Why were these changed to using the `+` syntax?

Reply:
Leaving these with the `${url.protocol}` causes eslint to fail.  I don't like the change, but made it to move forward with the PR